Eshop.Domain/Category.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Eshop.Domain
+{
+    public class Category
+    {
+        public Category(string categoryName, int id, string description)
+        {
+            this.id = id;
+            this.description = description;
+        }
+        public string categoryName { get;  set; }
+        public int id { get; private set; }
```

Comment:
Here actually you are missing vertical space.

Reply:
I have a new Category.cs which should have vertical space in the right plcae

Reply:
edit - dont have new category.cs but its reworked and fixed